### DISCUSSION TITLE
lf icon color

### DISCUSSION THREAD

**Discussion by techeddie** [gokcehan/lf#1408] [General]
Hello,
I'm using LF_COLORS and LF_ICONS in my config but I would like to specify a specific directory icon color.
The directory name should not be changed in the same time. I tried to find some details in the docu.
Is this possible?
Thanks

**Comment by joelim-work** [comment]
Unfortunately `lf` currently doesn't support using a separate color for icons, there is already an open issue for this #928. I don't think anyone has looked into it much recently but feel free to add your own comments/suggestions if you have any.

**Reply by techeddie** [reply]
okay, thanks for info.. I placed my comment. have a nice one

**Comment by techeddie** [comment]
following: https://github.com/gokcehan/lf/issues/928
